### PR TITLE
CONN-1757 change method label from Authentication to Authorization

### PR DESCRIPTION
**Pull request by trantang**

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CallbackProperties.java [hunk 1]
```diff
@@ -87,19 +87,19 @@
...
-     * @return Authenication Statement Exists
+     * @return Authorization Statement Exists
      */
-    public Boolean getAuthenicationStatementExists();
+    public Boolean getAuthorizationStatementExists();
 
     /**
      * @return Authnication Resource
      */
     public String getAuthnicationResource();
```
Comment (TabassumJafri) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CallbackProperties.java]: this should be labeled as getAuthorizationResource

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java [hunk 2]
```diff
@@ -329,18 +329,18 @@ protected Conditions createConditions(final CallbackProperties properties) {
...
+    public List<AuthzDecisionStatement> createAuthorizationDecisionStatements(final CallbackProperties properties,
         final Subject subject) {
         final List<AuthzDecisionStatement> authDecisionStatements = new ArrayList<>();
 
-        final Boolean hasAuthzStmt = properties.getAuthenicationStatementExists();
+        final Boolean hasAuthzStmt = properties.getAuthorizationStatementExists();
         // The authorization Decision Statement is optional
         if (hasAuthzStmt) {
             // Create resource for Authentication Decision Statement
             final String resource = properties.getAuthnicationResource();
```
Comment (TabassumJafri) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java]: the methos should be getAuthorizationResource

File: Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/opensaml/CallbackMapPropertiesTest.java [hunk 3]
```diff
@@ -109,8 +107,8 @@ public void setUp() {
     @Test
     public void testCallbackMapPropertiesGetters() {
         assertEquals(callbackProperties.getAssertionIssuerFormat(), TEST_VALUE);
-        assertEquals(callbackProperties.getAuthenicationDecision(), TEST_VALUE);
-        assertTrue(callbackProperties.getAuthenicationStatementExists());
+        assertEquals(callbackProperties.getAuthorizationDecision(), TEST_VALUE);
+        assertTrue(callbackProperties.getAuthorizationStatementExists());
         assertEquals(callbackProperties.getAuthenticationContextClass(), TEST_VALUE);
         assertEquals(callbackProperties.getAuthenticationSessionIndex(), TEST_VALUE);
         assertEquals(callbackProperties.getAuthnicationResource(), TEST_VALUE);
```
Comment (TabassumJafri) [inline Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/opensaml/CallbackMapPropertiesTest.java]: Fix the label here as well.

**Review by TabassumJafri — CHANGES_REQUESTED** [review]
Please see my inline comments

**Review by mhpnguyen — APPROVED** [review]

**Pull request merged**